cmd/spire-agent/cli/run/run.go

```diff
@@ -235,6 +241,10 @@ func (c *agentConfig) validate() error {
 		return errors.New("only one of trust_bundle_url or trust_bundle_path can be specified, not both")
 	}
 
+	if c.TrustBundleFormat != bundleFormatPEM && c.TrustBundleFormat != bundleFormatSPIFFE {
+		return errors.New("invalid value for trust_bundle_format, expected 'pem' or 'spiffe'")
```

Comment:
It would be good to reference the constants here.
`fmt.Sprintf` with `%q` can be used for that.